src/util/util.ts

```diff
@@ -65,7 +65,8 @@ export function getNftFromRaw(rawNft: RawNft, contractAddress: string): Nft {
     metadataError: rawNft.error,
     rawMetadata: rawNft.metadata,
     tokenUri: parseNftTokenUri(rawNft.tokenUri),
-    media: parseNftTokenUriArray(rawNft.media)
+    media: parseNftTokenUriArray(rawNft.media),
+    spamInfo: rawNft.spamInfo
```

Comment:
`isSpam` comes from the backend as a string, so we'll need to parse it to a boolean here.

Reply:
changed the raw type and parsing